sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json

```diff
@@ -501,6 +501,7 @@
     "S6863",
     "S6889",
     "S6905",
-    "S6909"
+    "S6909",
+    "S6914"
```

Comment:
Should we keep this rule in the Sonar Way? Let's double-check with @johann-beleites-sonarsource.
I think this is a valid rule, although forces to use Google API which may not be available on all devices.

Reply:
I agree: should not be in Sonar Way profile.